### PR TITLE
add admin-list-articles.hbs

### PR DESCRIPTION
**Pull request by cardona6**
add link for articles

### CODE REVIEW

File: server/views/category.hbs [hunk 1]
```diff
@@ -8,4 +8,8 @@
     <input type="text" name="shortDescription">
     <br>
     <input type="submit" value="Submit">
-</form>
\ No newline at end of file
+</form>
+
+<nav class="nav">
+  <a class="nav-link" href="#">Add an Article </a>
```
Comment (kabaros) [inline server/views/category.hbs:14]: the link should not be here - it should be in list

File: server/views/admin-list-articles.hbs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <div class="card-block">
+            <h4 class="card-title">article:{{this.title}}</h4>
+            <p class="card-text">{{this.fullContent}}</p>
+        </div>
+        </div>
+
+
+{{/each}}
+    </div>
+        <a href="article/add"><button class="bg-info"> Add New article</button></a>
```
Comment (kabaros) [inline server/views/admin-list-articles.hbs:19]: does this link work - I don't think so

**Review by kabaros — CHANGES_REQUESTED** [review]

File: server/routes/admin.js [hunk 3]
```diff
@@ -17,6 +17,15 @@ router.get('/categories', function (req, res, next) {
     categoryClient.listCategory(callback);
 });
 
+router.get('/articles', function (req, res, next) {
+  const callback = (error, article) => {
+     res.render("admin-list-articles",{
+      article
```
Comment (kabaros) [inline server/routes/admin.js:23]: the variable name should be articles, not article (because it's an array)

Comment (rarmatei) [reply server/routes/admin.js:23]: Also, I would recommend the long notation while we're still learning JavaScript:

```javascript
{
   articles: articles
}

File: server/views/add-articles.hbs [hunk 4]
```diff
@@ -9,4 +9,9 @@
...
     <button type="submit">Send</button>
 </form>
- 
\ No newline at end of file
+ 
+<hr>
+
+ <nav class="nav">
+  <a class="nav-link" href="#">Add an Article </a>
+</nav>
```
Comment (rarmatei) [inline server/views/add-articles.hbs:17]: What was the reason for adding this nav at the bottom?

I think we can just revert this file?

File: server/views/admin-list-articles.hbs [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <div class="row justify-content-md-center">
+
+     
+
+
```
Comment (rarmatei) [inline server/views/admin-list-articles.hbs:6]: We can format this document a bit better and remove all this necessary whitespace (both above and below)

File: server/views/admin-list-articles.hbs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    {{#each article}}
+
+        <div class="card" style="width: 20rem;">
+        <div class="card-block">
+            <h4 class="card-title">article:{{this.title}}</h4>
+            <p class="card-text">{{this.fullContent}}</p>
+        </div>
```
Comment (rarmatei) [inline server/views/admin-list-articles.hbs:13]: This inner div can be indented more, since its a child of the outer div

**Review by rarmatei — CHANGES_REQUESTED** [review]

**Pull request closed without merging**